src/AutoRest.CSharp.V3/Generation/Writers/CodeWriter.cs

```diff
@@ -269,13 +269,13 @@ public CodeWriter Literal(object? o)
         public CodeWriter Line(FormattableString formattableString)
         {
             Append(formattableString);
-            Line();
+            _builder.AppendLine();
```

Comment:
This needs to always append a line because you are adding text (not just a blank line).

Reply:
But your logic should be able to handle it just fine.

Reply:
The condition for writing a new line in this situation is different than `Line()`. If any text is written during `Append(formattableString)`, it should always write a new line. The reason this is for a situation where the line we appended has the last character as `{`. Obviously, we want a new line after '{'. The normal `Line()` method is for blank line insertion (no appended text). So, what I did here now is check to see if `formattableString` has text (which allows us to know that text was written). This was significantly easier and cleaner than changing `Append(formattableString)` to 'let us know text was written'.

What I've also done is added `LineRaw()` (which always writes a new line) to follow the pattern we used for the rest of the raw methods. Then, I call `LineRaw()` here and in the `Line()` method (when the condition is met).